graph-manager-docs/source/schema-registry.mdx

```diff
@@ -33,15 +120,27 @@ id      schema        tag
 190330  example-4218  current
 ```
 
-The CLI fetches your schema via a configuration option you set up in [Prerequisites](#prerequisites) and pushes it to Graph Manager.
+The CLI fetches your schema via a configuration option you set up in the project configuration above and pushes it to Graph Manager.
 
-Every time you push a new version of your schema, it's appended to your graph's schema history.
+### Registering manually to a variant
 
-## Registering a schema via continuous delivery
+To register a schema to a particular [variant of your graph](#managing-environments-with-variants), include the `--variant=<VARIANT>` flag in your `apollo service:push` command, like so:
 
-To get the most out of Graph Manager, you should **register each update to your production schema as soon as it occurs**. To achieve this, include a call to `apollo service:push` in your continuous delivery pipeline.
+```bash
+apollo service:push --variant=beta
+```
+
+If you omit the `--variant` flag, the `apollo service:push` command pushes to the default graph variant, named `current`.
+
+## Registering with continuous delivery
+
+To get the most out of Graph Manager, you should **register each update to your production schema as soon as it occurs**. Consequently, schema registration should be part of your continuous delivery pipeline.
+
+**If you're using [automatic schema reporting](#automatic-registration)** with Apollo Server, registration happens automatically after your server's deployment completes. No additional continuous delivery configuration is required.
```

Comment:
Can we specifically say this is the preferred method? I always prefer, when specifying multiple options, to point out which is the recommended way